src/interactors/interactors.test.js

```diff
@@ -0,0 +1,32 @@
+import Cloud, * as cloudMocks from '@cesarbr/knot-fog-connector-knot-cloud';
+import RegisterDevice from './RegisterDevice';
+
+jest.mock('@cesarbr/knot-fog-connector-knot-cloud');
+
+const mockThing = {
+  id: 'abcdef1234568790',
+  name: 'my-device',
+};
+
+describe('Interactors', () => {
+  beforeEach(() => {
+    cloudMocks.mockAddDevice.mockClear();
+    cloudMocks.mockRemoveDevice.mockClear();
+    cloudMocks.mockUpdateSchema.mockClear();
+    cloudMocks.mockPublishData.mockClear();
+  });
+
+  test('registerDevice: should execute interactor correctly when register a new thing on cloud', async () => {
```

Comment:
Grammar: the word `register` should be in gerund (end with a `-ing`), since we're referring to it as an action :point_down: 
```suggestion
  test('registerDevice: should execute interactor correctly when registering a new thing on cloud', async () => {
```

Reply:
Done!